### ISSUE TITLE
Groom Chatbot KPIs for Product Success Tool

### ISSUE THREAD

**Issue by joanneesteban** [department-of-veterans-affairs/va.gov-team#9731]
## User Story or Problem Statement
How might we understand the requirements for adding VAOS KPIs to Domo so that when we start building on Domo, we can share which metrics we can add, which metrics will have a BigQuery or a GTM dependency, and how we can proceed with building?

## Goal
Determine efforts and sources of provided KPIs.

---
## Tasks
- [x] Determine data sources for each KPI
- [x] Determine dependencies for KPIs (BigQuery or GTM)
- [x] Document available vs. unavailable metrics
- [x] If any, add implementation issue for additional necessary implementation

## Acceptance Criteria
- [ ] VSP Analytics has sources & dependencies for provided KPIs

---

## Links
[Chatbot KPIs](https://github.com/department-of-veterans-affairs/va.gov-team/issues/9714#issue-629349759)

Current Chatbot data implementation issues:
https://github.com/department-of-veterans-affairs/va.gov-team/issues/9611
https://github.com/department-of-veterans-affairs/va.gov-team/issues/9621
https://github.com/department-of-veterans-affairs/va.gov-team/issues/9622

**Comment by joanneesteban** [comment]
@Kimberley2019 adding you to this issue because there are some KPIs that have Contact Centers as the source. We'll need to find where that data is (probably in the Covid dashboard, but document that for this purpose).

**Comment by bsmartin-ep** [comment]
@joanneesteban / @jonwehausen -

New cards added to [COVID Chatbot dashboard](https://va-gov.domo.com/page/62901069) at their request...

KPI 1:

- [Inbound traffic to chatbot from external sources with greater than 5 users](https://va-gov.domo.com/page/62901069/kpis/details/472636070)

KPI 3:

- [VEO COVID calls vs COVID Chatbot Pageviews](https://va-gov.domo.com/page/62901069/kpis/details/237622102)
- [COVID FAQ Pageviews vs COVID Chatbot Pageviews](https://va-gov.domo.com/page/62901069/kpis/details/1823539260)

@jonwehausen  - 

Any DOMO way to remove / ignore those VETText outliers in the KPI 3 cards?  I tried the outlier filtering in the graph, but it just hides the entire line.

**Comment by bsmartin-ep** [comment]
From https://dsva.slack.com/archives/C1Q4294JJ/p1593625468221100...

> **Ramya Ramesh** 
> Hi Joanne / Jon Wehausen - We have a request regarding the Chatbot KPI Dashboard.
> Right now, we are able to track how many folks clicked on a phone number from the chatbot page. We would like to extend this insight and are wondering if it is possible to also see what percentage of mobile sessions (that make to a screen with a clickable phone number) end up with clicking a number?
> 
> This perhaps might involve some way of segregating the sessions into ones that reached a clickable phone number and ones that did not.
> 
> Please let us know your thoughts on what changes this might entail for us.

Today I am creating a custom segment in GA and a DOMO dataset to attempt to get this insight.

**Comment by joanneesteban** [comment]
To Do: For any charts that don't have dates shown, specify the date ranges either in notes or in the title. (i.e. April 13 - present).

**Comment by bsmartin-ep** [comment]
re:  https://github.com/department-of-veterans-affairs/va.gov-team/issues/9731#issuecomment-655599511

> We would like to extend this insight and are wondering if it is possible to also see what percentage of mobile sessions (that make to a screen with a clickable phone number) end up with clicking a number?

Added these cards to (using this [dataset](https://va-gov.domo.com/datasources/e274ffbe-2a72-4774-869e-eaf874c7825a/details/overview)) to [Chatbot KPI dashboard](https://va-gov.domo.com/page/62901069?userId=2108773409).  Note that this data is at a user scope and not a session scope as originally requested due to GA limitations. 

![image](https://user-images.githubusercontent.com/62304138/87225847-e2a41380-c35d-11ea-999a-45681b023202.png)

**Comment by joanneesteban** [comment]
Additional AC:

- [ ] Dynamic or asterisk on date ranges for filtered data

**Comment by joanneesteban** [comment]
Original request completed. Waiting on API requests from Aaron and will open a new ticket when that comes in.
